pkg/entity-store/store.go

```diff
@@ -29,6 +29,11 @@ const (
 	// should wait until the READY state to use the object
 	StatusCREATING Status = "CREATING"
 
+	// StatusINTRANSIT object is currently being processed
+	// and should NOT be picked up by periodic sync
+	// TODO(imikushin) GC stale inflight objects - in case responsible process terminates (e.g. using process UUIDs)
+	StatusINTRANSIT Status = "INTRANSIT"
```

Comment:
We talked about this.  This isn't a good status as it doesn't imply what transition is happening, we lose context.  I'm not going to hold back this PR, but this isn't a pattern I'd like to push through dispatch.

Also, it doesn't convey any meaning to the user.  The actual status should be "CREATING" which means it is in transition from INITIALIZED -> READY.  The function could be created with the INITIALIZED initial state (like images)